### DISCUSSION TITLE
[5.2] Drag and move file in media manager

### DISCUSSION THREAD

**Discussion by micker** [joomla/joomla-cms#47048] [2. Cold Features]
New media manager is realy good ! but need a little polishing
is it possible to add drag and move file in folder ?
![image](https://github.com/joomla/joomla-cms/assets/2539440/2f0c1cc2-7cff-40cf-8104-73333b3032d6)
regards

**Comment by dgrammatiko** [comment]
> is it possible to add drag and move file in folder ?

Sort answer: yes it's possible, using https://github.com/hcg1023/vue3-dnd

Reality check: the media manager needs to be upgraded to composition, which is a bit more work (FWIW I started that some time ago but gave up, it's too much work: https://github.com/joomla/joomla-cms/pull/41542)

**Comment by micker** [comment]
ok i understand

**Comment by Fedik** [comment]
> the media manager needs to be upgraded to composition

Actually it doesn't matter. 
What is really matter, who would like (and have a time) to work on the feature.
